tests/MongoDB.Driver.Core.TestHelpers/CoreTestConfiguration.cs

```diff
@@ -227,15 +228,18 @@ public static CollectionNamespace GetCollectionNamespaceForTestMethod(string cla
 
         private static ConnectionString GetConnectionString()
         {
-            var uri = Environment.GetEnvironmentVariable("MONGODB_URI") ?? Environment.GetEnvironmentVariable("MONGO_URI");
-            if (uri == null)
-            {
-                uri = "mongodb://localhost";
-                if (IsReplicaSet(uri))
-                {
-                    uri += "/?connect=replicaSet";
-                }
-            }
+            // TODO: move this to EG config level
+            var uri = "mongodb://localhost:17017?loadBalanced=true";
```

Comment:
this is steps to configure LB:

    How to run loadBalancing on windows:
1. Download nginx: http://nginx.org/en/download.html
2. Update nginx-1.21.0\conf\nginx.conf on:

            #user  nobody;
	        worker_processes  1;
	        #error_log  logs/error.log;
	        #error_log  logs/error.log  notice;
	        #error_log  logs/error.log  info;
	        # pid        nginx/logs/nginx.pid;
	        events {
		     worker_connections  1024;
	        }
	       stream {
		   server {
		       listen 17017;
		       proxy_pass single_server_backend;
		    }
       		    server {
		       listen 17018;
 		       proxy_pass multi_server_backend;
		    }
		    upstream multi_server_backend {
			   server localhost:27017;
			   server localhost:27018;
		    }
		    upstream single_server_backend {
			   server localhost:27017;
		    }    
	     }
	
pay attention on ports 17017 and 17018 that represents single mongos and two mongos accordingly.
So, to use them a driver should consume the following connection strings: mongodb://localhost17017?loadBalanced=true or mongodb://localhost17018?loadBalanced=true
3.
cd c:\
unzip nginx-1.21.0.zip
cd nginx-1.21.0
start nginx
4. Verify that nginx has been successfully launched:
C:\nginx-1.21.0>tasklist /fi "imagename eq nginx.exe"
Image Name                     PID Session Name        Session#    Mem Usage
========================= ======== ================ =========== ============
nginx.exe                     1680 Console                    1     14,188 K
nginx.exe                    14920 Console                    1     14,356 K
5. Run server with 2 mongos:
mlaunch init --sharded 1 --replicaset --mongos 2 --setParameter enableTestCommands=1